Sim/SimPapas/src/PapasExportParticlesTool.cpp

```diff
@@ -1,14 +1,21 @@
 #include "PapasExportParticlesTool.h"
+// FCCSW
 #include "datamodel/GenVertex.h"
 #include "datamodel/LorentzVector.h"
+//PAPAS
+#include "papas/datatypes/HistoryHelper.h"
+#include "papas/datatypes/IdCoder.h"
+
 
 DECLARE_TOOL_FACTORY(PapasExportParticlesTool)
 
 PapasExportParticlesTool::PapasExportParticlesTool(const std::string& aType, const std::string& aName,
                                                    const IInterface* aParent)
     : GaudiTool(aType, aName, aParent) {
-  declareInterface<IPapasTool>(this);
+  declareInterface<IPapasExportTool>(this);
   declareProperty("recparticles", m_particlesHandle);
+  declareProperty("genparticles", m_iGenpHandle);
```

Comment:
the name is not symmetric with the line just above. why not m_genParticlesHandle?

Reply:
Yes indeed! thanks.